packages/web-components/fast-foundation/src/picker/picker-list.ts

```diff
@@ -0,0 +1,54 @@
+import { attr } from "@microsoft/fast-element";
+
+import { FoundationElement } from "../foundation-element";
+
+/**
+ * A List Picker Menu Custom HTML Element.
+ *
+ * @alpha
+ */
+export class PickerList extends FoundationElement {
+    /**
+     * Applied to the aria-label attribute of the input element
+     *
+     * @alpha
+     * @remarks
+     * HTML Attribute: label
+     */
+    @attr({ attribute: "label" })
+    public label: string;
+
+    /**
+     * Applied to the aria-labelledby attribute of the input element
+     *
+     * @alpha
+     * @remarks
+     * HTML Attribute: labelledby
+     */
+    @attr({ attribute: "labelledby" })
+    public labelledby: string;
+
+    /**
+     * The input element of the component
+     *
+     * @alpha
```

Comment:
should this be internal?

Reply:
The parent `picker` component uses the reference - `this.listElement.inputElement.value`

Reply:
Is exposing this element necessary? Is it possible to just implement a proxy property off the custom element that resolves the internal element's value property? I see you're adding event listeners directly as well - can these just listen on the host custom element?

I'm asking for a two reasons:
1. Where this property comes from seems like an implementation detail of the custom element, so it seems a little odd to me to have to reach onto an shadowed form element to get the value.
2. Leaking an element out of the shadow DOM provides an easy mechanism to obtain access to the shadowRoot and other elements in the shadow DOM, which effectively prohibits authors desiring a closed shadow root the ability to effectively do so.

Reply:
Reworking this, picker-list doesn't need to be its own component and would be better as a simple template that picker instances internally.  Seems better and lighter too.

Reply:
Changed this to a more slot oriented approach.